packages/app-store/zohocrm/lib/CrmService.ts

```diff
@@ -74,7 +76,7 @@ export default class ZohoCrmCrmService implements CRM {
     });
     const response = await axios({
       method: "post",
-      url: `https://www.zohoapis.com/crm/v3/Contacts`,
+      url: `https://www.zohoapis.${this.location}/crm/v3/Contacts`,
```

Comment:
Now, where do we retrieve the location? I saw previously that you set it in the callback. For example: `const server_location = location === "us" ? "com" : location`;

Reply:
Yeah I didn't finished, I think we will need a bigger refactoring since there is not only the zohocrm app that could be affected right?